prod_env/model_deployments.yaml

```diff
@@ -0,0 +1,19 @@
+# This file defines all the model deployments that are supported by the Helm API.
+# Some models have several deployments, each with different parameters.
+
+# If you want to add a new deployment, you can technically do it here but we recommend
+# you to do it in prod_env/model_deployments.yaml instead.
+
+# Follow the template of this file to add a new deployment. You can copy paste this to get started:
+#    # This file defines all the model deployments that you do not want to be public.
+#    model_deployments: [] # Leave empty to disable private model deployments
+
+model_deployments:
+  - name: cohere/command-r
+    model_name: cohere/command-r
+    tokenizer_name: cohere/cohere
+    max_sequence_length: 2048
+    client_spec:
+      class_name: "helm.clients.cohere_client.CohereClient"
+    window_service_spec:
```

Comment:
Related to later note about tokenizers: if you use the Hugging Face tokenizer rather than the Cohere tokenizer API, then you can just delete `window_service_spec` (which exists to work around the length limit of the Cohere tokenizer API), which will make HELM use the default window service.